plugins/commands/snapshot/command/save.rb

```diff
@@ -28,7 +33,19 @@ def execute
 
           name = argv.pop
           with_target_vms(argv) do |vm|
-            vm.action(:snapshot_save, snapshot_name: name)
+            # TODO: remove this comment before merge....
+            # do I need to check if the provider on the vm supports snapshots before doing this?
+            snapshot_list = vm.provider.capability(:snapshot_list)
```

Comment:
@chrisroberts question about this here...should I check to see if the vms provider has the capability to take snapshots before calling this action? I noticed this step happens else where in a different snapshot command.